src/styles/blocks/container.scss

```diff
@@ -0,0 +1,22 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+
+  @media (min-width: 1200px) {
+    padding: 50px 128px;
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/88807331/179546840-52bf8df3-7f73-45f7-9028-6ec64ef8e7ab.png)
Elements must be in 4 columns at wide resolution. Try to calculate padding based on screen width or try to make 2 row containers and put 4 cards in each.